util/chunk/row_container_test.go

```diff
@@ -30,15 +30,15 @@ func (r *rowContainerTestSuite) TestNewRowContainer(c *check.C) {
 	fields := []*types.FieldType{types.NewFieldType(mysql.TypeLonglong)}
 	rc := NewRowContainer(fields, 1024)
 	c.Assert(rc, check.NotNil)
-	c.Assert(rc.AlreadySpilled(), check.Equals, false)
+	c.Assert(rc.AlreadySpilledSafe(), check.Equals, false)
 }
 
 func (r *rowContainerTestSuite) TestSel(c *check.C) {
```

Comment:
Could you add some tests for `SortedRowContainer`?

Reply:
Addressed.